applications/audio/broadcast_configuration_tool/src/main.c

```diff
@@ -0,0 +1,504 @@
+
+static struct bt_bap_lc3_preset lc3_preset_24_2_2 = BT_BAP_LC3_BROADCAST_PRESET_24_2_2(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_32_2_1 = BT_BAP_LC3_BROADCAST_PRESET_32_2_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_32_2_2 = BT_BAP_LC3_BROADCAST_PRESET_32_2_2(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_2_1 = BT_BAP_LC3_BROADCAST_PRESET_48_2_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_2_2 = BT_BAP_LC3_BROADCAST_PRESET_48_2_2(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_4_1 = BT_BAP_LC3_BROADCAST_PRESET_48_4_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_4_2 = BT_BAP_LC3_BROADCAST_PRESET_48_4_2(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_6_1 = BT_BAP_LC3_BROADCAST_PRESET_48_6_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_6_2 = BT_BAP_LC3_BROADCAST_PRESET_48_6_2(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+struct bap_preset {
+	struct bt_bap_lc3_preset *preset;
+	char name[PRESET_NAME_MAX];
+};
+
+static struct bap_preset bap_presets[] = {
+	{.preset = &lc3_preset_8_2_1, .name = "8_2_1"},
+	{.preset = &lc3_preset_8_2_2, .name = "8_2_2"},
+	{.preset = &lc3_preset_16_2_1, .name = "16_2_1"},
+	{.preset = &lc3_preset_16_2_2, .name = "16_2_2"},
+	{.preset = &lc3_preset_24_2_1, .name = "24_2_1"},
+	{.preset = &lc3_preset_24_2_2, .name = "24_2_2"},
+	{.preset = &lc3_preset_32_2_1, .name = "32_2_1"},
+	{.preset = &lc3_preset_32_2_2, .name = "32_2_2"},
+	{.preset = &lc3_preset_48_2_1, .name = "48_2_1"},
+	{.preset = &lc3_preset_48_2_2, .name = "48_2_2"},
+	{.preset = &lc3_preset_48_4_1, .name = "48_4_1"},
+	{.preset = &lc3_preset_48_4_2, .name = "48_4_2"},
+	{.preset = &lc3_preset_48_6_1, .name = "48_6_1"},
+	{.preset = &lc3_preset_48_6_2, .name = "48_6_2"},
+};
+
+/**
+ * @brief	Broadcast source static extended advertising data.
+ */
+static struct broadcast_source_ext_adv_data ext_adv_data[] = {
+	{.uuid_buf = &uuid_data,
+	 .pba_metadata_vacant_cnt = BROADCAST_SOURCE_PBA_METADATA_VACANT,
+	 .pba_buf = pba_data[0]},
+	{.uuid_buf = &uuid_data2,
+	 .pba_metadata_vacant_cnt = BROADCAST_SOURCE_PBA_METADATA_VACANT,
+	 .pba_buf = pba_data[1]}};
+#else
```

Comment:
Knowledge Sharing: Will we ever want to use this as not an Auracaster? What is the difference between a broadcaster that's an Auracaster and one that's not?

Reply:
It depends on how the adv data is structured and it also has to support 16kHz. I guess SIG will be the most happy if everything is Auracasters